src/Leaflet.VectorGrid.js

```diff
@@ -5,6 +5,7 @@ L.VectorGrid = L.GridLayer.extend({
 	options: {
 		rendererFactory: L.svg.tile,
 		vectorTileLayerStyles: {},
+		pointFeatureHooks: [],
```

Comment:
I would prefer this to be `oneachfeature`, to align better with `L.GeoJSON` - it's essentially the same functionality.

Also, I'd prefer to keep it simple and allow _one_ hook - again, keep things similar to existing API to avoid surprises.